### ISSUE TITLE
Nerf Knightmetal in Tinkers' Construct

### ISSUE THREAD

**Issue by monodemono** [DarkPacks/SevTech-Ages#3171]
Knightmetal's stats are outrightly superior to everything else available for a long time in every way, while being available as early as age 2. The stats need to be adjusted.

**Comment by ByFe** [comment]
I totally agree with that, it's pretty OP in early ages when you get some of the helmet crab spawners to your home within a sack of holding and get infinite shards this way, but you must fight your way through the stronghold first anyway.
